src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1556,12 +1555,15 @@ def delete_app_settings(cmd, resource_group_name, name, setting_names, slot=None
 # TODO: Centauri currently return wrong payload for update appsettings, remove this once backend has the fix.
     if centauri_functionapp:
         update_application_settings_polling(cmd, resource_group_name, name, app_settings, slot, client)
-        result = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'list_application_settings', slot)
+        _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'list_application_settings', slot)
     else:
-        result = _generic_settings_operation(cmd.cli_ctx, resource_group_name, name,
-                                             'update_application_settings',
-                                             app_settings, slot, client)
-    return _build_app_settings_output(result.properties, slot_cfg_names.app_setting_names if slot_cfg_names else [])
+        _generic_settings_operation(cmd.cli_ctx, resource_group_name, name,
+                                    'update_application_settings',
+                                    app_settings, slot, client)
+
+    # logging as warning for success status message
+    logger.warning("Successfully deleted %s. Use config appsettings list command to view app settings.",
```

Comment:
I'm curious why we use "warning" instead of "info" for success status.

Reply:
INFO log is only displayed when `--verbose` or `--debug` is provided. If none of them is provided, INFO log won't be shown.